index.html

```diff
@@ -417,8 +423,16 @@ <h2><code>PointerEvent</code> Interface</h2>
 
                 <p>The <dfn><code>PointerEventInit</code></dfn> dictionary is used by the <dfn><code>PointerEvent</code></dfn> interface's constructor to provide a mechanism by which to construct untrusted (synthetic) pointer events. It inherits from the {{MouseEventInit}} dictionary defined in [[UI-EVENTS]]. The steps for constructing an event are defined in [[DOM]]. See the <a href="#examples" title="examples">examples</a> for sample code demonstrating how to fire an untrusted pointer event.</p>
 
-                <div class-"note">Pointer Events include two complementary sets of attributes to express the orientation of a transducer relative the X-Z plane: <dfn><code>tiltX</code></dfn> / <dfn><code>tiltY</code></dfn> (introduced in the original Pointer Events specification), and <dfn><code>altitudeAngle</code></dfn> / <dfn><code>azimuthAngle</code></dfn> (adopted from the <a href="https://w3c.github.io/touch-events/">Touch Events - Level 2</a> specification).
-                    Implementations MUST support both sets of attributes. When an untrusted (synthetic) Pointer Event is created programmatically using the constructor, and only one set of values is provided, the complementary set of attributes MUST be calculated and initialized.</div>
+                <div class="note">
+                    Pointer Events include two complementary sets of attributes to express the orientation of a transducer relative to the X-Y plane: <dfn><code>tiltX</code></dfn> / <dfn><code>tiltY</code></dfn> (introduced in the original Pointer Events specification), and <dfn><code>altitudeAngle</code></dfn> / <dfn><code>azimuthAngle</code></dfn> (adopted from the <a href="https://w3c.github.io/touch-events/">Touch Events - Level 2</a> specification).
+                    Implementations MUST support both sets of attributes. When an untrusted (synthetic) Pointer Event is created programmatically using the constructor, and only one set of attributes is provided, the complementary set of attributes MUST be calculated and initialized by the user agent. If both sets of attributes are provided, no calculation should be performed.
+
+                    <p>
```

Comment:
I have two editorial suggestions about this para about rounding:

1. This para also applies also to trusted events (when UA fills in missing values from device driver).  But the next para as well as the last two sentences in the above para is about untrusted events only.  I think it would be easier to read if we move this para to the end, and have a separate para in the middle for untrusted events.  Wdyt?

2. Let's omit the standard rounding rule here, instead refer to `Math.round()`: https://tc39.es/ecma262/#sec-math.round. I suggest the following to cover both untrusted and trusted cases:
> When the user agent calculates tiltX/tiltY from azimuth/altitude, it SHOULD round the final integer values using the Math.round rules [ref].

Reply:
Done